password_security/static/src/js/password_gauge.js

```diff
@@ -6,22 +6,6 @@ odoo.define('password_security.policy', function (require) {
     var core = require('web.core');
     var _t = core._t;
     var Policy = require('auth_password_policy').Policy;
```

Comment:
Do not require the module twice.

```suggestion
     var auth_password_policy = require('auth_password_policy');
     var Policy = auth_password_policy.Policy;
```